---
title: Gamma Interferon-Dependent Transcriptional Memory via Relocalization of a Gene Locus to PML Nuclear Bodies
authors: Manolis Gialitakis, Panagiota Arampatzi, Takis Makatounakis, Joseph Papamatheakis
year: 2010
tags:
 - transcription 
 - antigen_presentation
---
# Gamma Interferon-Dependent Transcriptional Memory via Relocalization of a Gene Locus to PML Nuclear Bodies   
Published 2010 in Molecular and Cellular Biology


## Abstract
 
 Memory of past cellular responses is an essential adaptation to repeating environmental stimuli. We addressed the question of whether gamma interferon (IFN-γ)-inducible transcription generates memory that sensitizes cells to a second stimulus. We have found that the major histocompatibility complex class II gene DRA is relocated to promyelocytic leukemia (PML) nuclear bodies upon induction with IFN-γ, and this topology is maintained long after transcription shut off. Concurrent interaction of PML protein with mixed-lineage leukemia generates a prolonged permissive chromatin state on the DRA gene characterized by high promoter histone H3 K4 dimethylation that facilitates rapid expression upon restimulation. We propose that the primary signal-induced transcription generates spatial and epigenetic memory that is maintained through several cell generations and endows the cell with increased responsiveness to future activation signals.


---
## key points:
- central claim is that the restimulation with [[type 2 ifn|ifngamma]] leads to earlier and stronger [[MHCII|MHC2]] indcution, demonstrating a mechanism of transcriptional memory 
-  this memory is created through sustained H3K4me2 mark, facilitated by the [[COMPASS_MLL|MLL complex]]s [[WDR5]] subunit interaction with [[PML]] isoform 4
- interestingly, as a contrast to innate immune memory,  H3K4me3 is not the driving modification, but instead [[H3K4me2]]

---


## Notes: 
- pre-stimulated cells show earlier and stronger expression regarding HLA-DR
	- pretreatment with ifngamma
- this difference is not driven by CIITA differences, as qPCR data shows similar levels 
- while [[histone 3]] acetylation is the same between conditions, [[H3K4me3]] and [[CIITA]] binding ist faster and stronger in theifn primed condition
- SspI, a restriction enzyme has a cutting site between the X and the Y box of MHC2, which only becomes accessible during transcription
	- the site is accessible after priming and during stimulation, but not in resting cells
- [[nucleosome]] depletion can be seen in primed and actively stimulated cells, but not in resting cells 
- [[CIITA]] levels, histone acetylation, for both histone 3 and histone 4, and H3K4me3 all returned to baseline levels 24h after ifn stimulation 
- [[H3K4me2]] persists for days and multiple cell generations 
- visual colocalisation of PML bodies and HLA-DRA gene was found
- [[COMPASS_MLL|MLL complex]] is associated with [[PML]] bodies after IFN stimulus
- the increase in HLA-DRA expression is dependant on PML expression, with an increase in PML leading to an increase in HLA-DRA expression
	- PML isoform 4 is associated with the locus and MLL 
- depletion of PML leads to a reduction of the memory effect
---